<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>450</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Logchain Monitoring </string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../Users/USER/Desktop/monitor.png</normaloff>../../../Users/USER/Desktop/monitor.png</iconset>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(40, 40, 40)</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>661</width>
     <height>5</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(44, 132, 238)</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>241</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget_2">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>170</y>
     <width>181</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget_3">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>170</y>
     <width>181</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>611</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>56</width>
     <height>12</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;</string>
   </property>
   <property name="text">
    <string>Node List</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;</string>
   </property>
   <property name="text">
    <string>Network Monitoring</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>621</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>함초롬바탕 확장</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
</string>
   </property>
   <property name="text">
    <string>Logchain Monitoring Board</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>5</width>
     <height>451</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(44, 132, 238)</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_3" native="true">
   <property name="geometry">
    <rect>
     <x>645</x>
     <y>0</y>
     <width>5</width>
     <height>451</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(44, 132, 238)</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_4" native="true">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>445</y>
     <width>661</width>
     <height>5</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(44, 132, 238)</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_5" native="true">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>410</y>
     <width>381</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(44, 132, 238)</string>
   </property>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>361</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>휴먼모음T</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;</string>
    </property>
    <property name="text">
     <string>Server Status : GOOD              14:22:39</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>150</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;</string>
   </property>
   <property name="text">
    <string>Blocks</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>150</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;</string>
   </property>
   <property name="text">
    <string>Transactions </string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
